gateway/src/apicast/policy/http_proxy/apicast-policy.json

```diff
@@ -0,0 +1,27 @@
+{
+  "$schema": "http://apicast.io/policy-v1/schema#manifest#",
+  "name": "Proxy service",
+  "summary": "Adds an HTTP proxy to the service.",
+  "description": [
+    "With this policy all the traffic for this service will be routed accross ",
+    "the defined proxy"
+  ],
+  "version": "builtin",
+  "configuration": {
+      "type": "object",
+      "properties": {
+        "all_proxy": {
```

Comment:
I took care in the code, also added a unittest. 
```
it("all_proxy does not overwrite http/https proxies", function()
    local proxy = proxy_policy.new({
      all_proxy = all_proxy_val,
      http_proxy = http_proxy_val,
      https_proxy = https_proxy_val
    })
    local callback = proxy:export()

    assert.same(callback.has_http_proxy(_, http_uri), resty_url.parse(http_proxy_val))
    assert.same(callback.has_http_proxy(_, https_uri), resty_url.parse(https_proxy_val))
  end)
```

Reply:
:+1:
I think the description of `all_proxy` or the description above should mention this. Otherwise, the users will not be aware of this when looking at the form to configure the policy.